docs/operations/hotplug.md

```diff
@@ -0,0 +1,129 @@
+# Hotplug Volumes
+
+KubeVirt now supports hotplugging volumes into a running Virtual Machine Instance (VMI). The volume must be either a block volume or contain a disk image file just like any other regular volume. When a VM that has hotplugged volumes is rebooted, the hotplugged volumes will NOT be attached to the restarted VM unless the volumes are persisted.
+
+## Enable feature gate
+
+In order to enable the HotplugVolumes feature gate, you must add the HotplugVolumes to the list of enabled featureGates.
+
+```yaml
+spec:
+  configuration:
+    developerConfiguration:
+      featureGates:
+      - HotplugVolumes
+```
+
+Once the feature gate is enabled, you will be able to hotplug volumes into a running VMI.
```

Comment:
For consistency, I'd link to https://github.com/kubevirt/user-guide/blob/master/docs/operations/activating_feature_gates.md, as done throughtout the rest of the docs - for instance in https://github.com/kubevirt/user-guide/blob/master/docs/operations/live_migration.md#enabling-the-live-migration-support .

Reply:
Done